### ISSUE TITLE
Apple M1 Support ? 

### ISSUE THREAD

**Issue by gumush** [Inlife/nexrender-boilerplate#25]
sudo node start.js
Started local static server at port: 23234
adding default render-only-node licenses for After Effects at:
 - /Users/hikmetgumus/Documents/ae_render_only_node.txt
 - /Users/hikmetgumus/Documents/Adobe/ae_render_only_node.txt
checking After Effects command line renderer patch...
command line patch already is in place
testing onChange: render:setup
[-sczofMfEqUlsZANxL6Jm] setting up job...
[-sczofMfEqUlsZANxL6Jm] working directory is: /Users/hikmetgumus/Documents/AutomatedCreation/nexrender-boilerplate/temp/-sczofMfEqUlsZANxL6Jm
testing onChange: render:download
[-sczofMfEqUlsZANxL6Jm] downloading assets...
testing onChange: render:prerender
[-sczofMfEqUlsZANxL6Jm] applying prerender actions...
testing onChange: render:script
[-sczofMfEqUlsZANxL6Jm] running script assemble...
testing onChange: render:dorender
[-sczofMfEqUlsZANxL6Jm] rendering job...
[-sczofMfEqUlsZANxL6Jm] spawning aerender process: /Applications/Adobe After Effects 2021/aerender -project /Users/hikmetgumus/Documents/AutomatedCreation/nexrender-boilerplate/temp/-sczofMfEqUlsZANxL6Jm/nm05ae12.aet -comp main -output /Users/hikmetgumus/Documents/AutomatedCreation/nexrender-boilerplate/temp/-sczofMfEqUlsZANxL6Jm/result.mov -s 0 -e 3600 -r /Users/hikmetgumus/Documents/AutomatedCreation/nexrender-boilerplate/temp/-sczofMfEqUlsZANxL6Jm/nexrender--sczofMfEqUlsZANxL6Jm-script.jsx
[-sczofMfEqUlsZANxL6Jm] rendering took ~8.321 sec.
[-sczofMfEqUlsZANxL6Jm] writing aerender job log to: /Users/hikmetgumus/Documents/AutomatedCreation/nexrender-boilerplate/temp/aerender--sczofMfEqUlsZANxL6Jm.log
testing onChange: render:postrender
[-sczofMfEqUlsZANxL6Jm] applying postrender actions...
[-sczofMfEqUlsZANxL6Jm] starting action-encode action (ffmpeg)
[-sczofMfEqUlsZANxL6Jm] action-encode: input file /Users/hikmetgumus/Documents/AutomatedCreation/nexrender-boilerplate/temp/-sczofMfEqUlsZANxL6Jm/result.mov
[-sczofMfEqUlsZANxL6Jm] action-encode: output file /Users/hikmetgumus/Documents/AutomatedCreation/nexrender-boilerplate/temp/-sczofMfEqUlsZANxL6Jm/output.mp4
Unsupported architecture.

**Comment by inlife** [comment]
Good point, seems like the ffmpeg version used does not support M1. Do you maybe know are there supported versions, maybe we can just update the dependency?

**Comment by gumush** [comment]
Updating ffmpeg to 4.4.0 solved this. But it still has an error.

aerender version 18.4.1x4
PROGRESS: Launching After Effects...
aerender ERROR -1701: AEGetParamPt failed at line 918

**Comment by inlife** [comment]
This seems to be unrelated to nexrender, not sure why that happens.
